index.js

```diff
@@ -126,6 +128,89 @@ function invokePromisedTaskFunction(taskFunction, cumulusMessage, context) {
   });
 }
 
+/**
+ * Get granules from execution message.
+ *   Uses the order of precedence as defined by the cumulus/common/message
+ *   description.
+ *
+ * @param {Object} message - An execution message
+ * @returns {Array<Object>} - An array of granule ids
+ */
+function getMessageGranules(message) {
+  const granules = get(message, 'payload.granules')
```

Comment:
We only ever pull from one of these? Is there no event where we might pull granules from multiple sources?

Reply:
This was actually just updated in Cumulus core, so where this code is looking is accurate:

https://github.com/nasa/cumulus/pull/1313

Reply:
@markdboyd Actually this change https://github.com/nasa/cumulus/pull/1313/files#diff-2fdbb43ab43a3b0596451c3fa38ecbc9R373 doesn't reflect what's here now.  see L141-143.

Reply:
Oh, I missed those lines because I wasn't looking at the full diff. So we probably should restrict it to just reading from `payload.granules` or `cma.event.payload.granules`?

Reply:
No I don't think so. That is for creating records that would not get completed. I think it's different. We're just associating logs with granules so I think we could find them from anywhere.